documentation/puppetdb_connection.markdown

```diff
@@ -101,7 +101,7 @@ The default value is false.
 
 When using multiple `server_urls`, this flag can be set to `true` to cause commands to be sent to all configured PuppetDB instances.
 
-The default value is false.
+The default value is `false` when not specified, but Puppet Enterprise 2016.5 and later explicitly configures `command_broadcast` to `true`.
```

Comment:
How about:

When set to `true` in installations using multiple `server_urls`, commands are sent to all configured PuppetDB instances. 

In open source Puppet and PE versions earlier than 2016.5, the default setting is `false`. In PE 2016.5 and later, the default setting is `true`.